Add ability to specify system paths to @old_configure_options

--- a/build/moz.configure/old.configure	2016-07-04 12:03:37.503301144 -0400
+++ b/build/moz.configure/old.configure	2016-07-04 12:04:33.443147224 -0400
@@ -149,16 +149,19 @@
     def all_options(help):
         return list(options)
 
     return depends(prepare_configure, extra_old_configure_args, all_options,
                    *options)
 
 
 @old_configure_options(
+    '--libdir',
+    '--includedir',
+    '--datadir',
     '--cache-file',
     '--enable-accessibility',
     '--enable-address-sanitizer',
     '--enable-alsa',
     '--enable-android-omx',
     '--enable-android-resource-constrained',
     '--enable-approximate-location',
     '--enable-b2g-bt',
